README.md

```diff
@@ -1,26 +1,72 @@
+[![Promote charm](https://github.com/canonical/github-runner-webhook-router/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/github-runner-webhook-router/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+This charm provides a webhook router for self-hosted GitHub Actions runners. 
+It is designed to be used in conjunction with the [GitHub Runner](https://github.com/canonical/github-runner-operator) charm.
 
-Use links instead.
--->
+The charm was built using the [paas-charm](https://github.com/canonical/paas-charm)  approach and runs on Kubernetes.
 
-# github-runner-webhook-router
+The router is a Flask application that listens for incoming webhooks from GitHub and routes them 
+to the appropriate GitHub Runner charm application, which then spawns a runner to execute the job.
+It is a critical component for the "reactive" mode of the GitHub Runner Charm.
 
-> **Warning**
-Work in progress: This charm is still in development and not yet ready for use in production.
 
-The github-runner-webhook-router charm is a Juju charm that provides a webhook router for GitHub Actions self-hosted runners. 
-This charm is designed to be used in conjunction with the [github-runner-operator](https://github.com/canonical/github-runner-operator) charm.
 
 
-## Other resources
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more.
+For the GitHub Runner Webhook Router, this includes
 
-<!-- If your charm is documented somewhere else other than Charmhub, provide a link separately. -->
+* All the general and Flask-related features of [paas-charm](https://github.com/canonical/paas-charm)
+* Configuring the routing table
+* Configuring a webhook secret to improve the security of the webhook endpoint
 
-- [Contributing](CONTRIBUTING.md) <!-- or link to other contribution documentation -->
+For information on how to deploy, integrate and manage this charm, please see the official
+[GitHub Runner Webhook Router Documentation](https://charmhub.io/github-runner-webhook-router).
 
-- See the [Juju SDK documentation](https://juju.is/docs/sdk) for more information about developing and improving charms.
+
+## Get started
+As the charm is designed to be used in conjunction with the [GitHub Runner](https://github.com/canonical/github-runner-operator) charm,
+please see [How to set up reactive spawning](https://charmhub.io/github-runner/docs/how-to-reactive) to learn how to deploy the charm.
+
+The charm requires an integration with mongodb (either the machine or k8s charm), otherwise it will
+go into a blocked state.
```

Comment:
I would move this line to a section called `## Integrations` (after the "Basic operations" section) and also add a line saying something like

`For a complete list of integrations, see the [Charmhub documentation](https://charmhub.io/github-runner-webhook-router/integrations).`